packages/core/src/components/sections/BannerNewsletter/BannerNewsletter.tsx

```diff
@@ -1,27 +1,38 @@
-import BannerText from 'src/components/ui/BannerText'
-import Newsletter from '../Newsletter'
+import BannerText, { BannerTextProps } from 'src/components/ui/BannerText'
+import Newsletter, { NewsletterProps } from 'src/components/ui/Newsletter'
 import Section from '../Section'
 import styles from './section.module.scss'
 
-function BannerNewsletter() {
+function BannerNewsletter({
+  banner,
+  newsletter,
+}: {
+  banner: BannerTextProps
+  newsletter: NewsletterProps
+}) {
   return (
     <Section
       className={`${styles.section} section-banner-newsletter layout__content`}
     >
       <div data-fs-banner-newsletter>
         <BannerText
-          title="Get to Know Our Next Release"
-          caption="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam elit nisi, vehicula in turpis sit amet, posuere aliquam nisl. "
```

Comment:
What about adding these values as default in the `sections.json` respective props? also valid for the other props.
(The lorem ipsum we can use another text)